src/snooping.jl

```diff
@@ -179,8 +180,8 @@ function snoop_packages(
     direct_test_deps = test_dependencies(pkgs)
     missing_pkgs = not_installed(Types.PackageSpec[direct_test_deps...])
     if install && !isempty(missing_pkgs)
-        Pkg.API.add_or_develop(ctx, missing_pkgs, mode = :add)
-    else
+        Pkg.API.add(ctx, missing_pkgs)
```

Comment:
Are you sure this works across Julia versions? I'm pretty sure I was using `add_or_develop` because `add` alone didn't work...

Reply:
Possibly it doesn't, let me install Julia 1.0 and check...it definitely fails on 1.3, though.